src/configure/configure.ts

```diff
@@ -533,6 +535,16 @@ class Orchestrator {
         }
     }
 
+    private filterPipelinesByResourceType(pipelineMap: Map<string, PipelineTemplate[]>) {
+        if (this.inputs.targetResource.resource && this.inputs.targetResource.resource.id) {
+            pipelineMap.forEach((element) => {
+                if (element.length > 0 && element[0].targetType !== this.inputs.targetResource.resource.type) {
+                    pipelineMap.delete(element[0].label);
```

Comment:
I have seen deletion of an element from a map while iterating through the map causes problem in c#. is that not a problem in JS?

Reply:
Tested a couple os scenarios, worked fine.
Any particular kind of issue caused in C#, which you can mention, so that I'll check that?

Reply:
I think if failed in removing the element itself. So it seems it would be okay here.